pypeit/spectrographs/keck_deimos.py

```diff
@@ -1306,3 +1307,87 @@ def deimos_read_1chip(hdu,chipno):
     return data, oscan
 
 
+
```

Comment:
Should these be methods in pypeit_collate_1d?

Reply:
possibly, but they are tuned for DEIMOS right now,
e.g. detector numbers and number of detectors.

We might be able to generalize further if we support
future instruments.